<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KF_GUIClass</class>
 <widget class="QWidget" name="KF_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kalman Filter</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 48);</string>
  </property>
  <widget class="QGroupBox" name="configureBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 241, 241);</string>
   </property>
   <property name="title">
    <string>Configure</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>147</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="libraryLayout">
     <item>
      <widget class="QLabel" name="libraryLebel">
       <property name="styleSheet">
        <string notr="true">color: rgb(241, 241, 241);</string>
       </property>
       <property name="text">
        <string>Library</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="radioButtonLayout">
       <item>
        <widget class="QRadioButton" name="assemblyRadioButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(241, 241, 241);</string>
         </property>
         <property name="text">
          <string>Assembly</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="cppRadioButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(241, 241, 241);</string>
         </property>
         <property name="text">
          <string>C++</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(241, 241, 241);</string>
         </property>
         <property name="text">
          <string>Threads used:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="usedThreadsLebel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 122, 204);</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="threadNumberSlider">
       <property name="styleSheet">
        <string notr="true">selection-color: rgb(255, 255, 0);
alternate-background-color: rgb(255, 255, 0);
color: rgb(255, 255, 0);
border-color: rgb(255, 255, 0);</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>6</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>461</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="inputLayout">
    <item>
     <widget class="QLabel" name="inputFileLebel">
      <property name="styleSheet">
       <string notr="true">color: rgb(241, 241, 241);</string>
      </property>
      <property name="text">
       <string>Input file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputFileLineEdit">
      <property name="styleSheet">
       <string notr="true">color: rgb(241, 241, 241);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="browseButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(241, 241, 241);background-color: rgb(67, 67, 70);</string>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="executionBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 241, 241);</string>
   </property>
   <property name="title">
    <string>Execution</string>
   </property>
   <widget class="QPushButton" name="executeButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(241, 241, 241);
background-color: rgb(67, 67, 70);</string>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plotButton">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(241, 241, 241);background-color: rgb(67, 67, 70);</string>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>81</width>
      <height>53</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Assembly</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>C++</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Execution times:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>90</width>
      <height>15</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="statusLayout">
     <item>
      <widget class="QLabel" name="statusTextLebel">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusLebel">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string>Ready</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="KF_GUI.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>browseSlot()</slot>
  <slot>exitSlot()</slot>
  <slot>returnedPressedSlot()</slot>
 </slots>
</ui>
